cmd/internal/cli/cache_clean_linux.go

```diff
@@ -40,15 +40,15 @@ var cacheCleanTypesFlag = cmdline.Flag{
 	DefaultValue: []string{"blob"},
 	Name:         "type",
 	ShortHand:    "T",
-	Usage:        "clean cache type, choose between: library, oci, and blob",
+	Usage:        "clean cache type, choose between: library, oci, shub, and blob",
 	EnvKeys:      []string{"TYPE"},
 }
 
 // -N|--name
 var cacheCleanNameFlag = cmdline.Flag{
 	ID:           "cacheCleanNameFlag",
 	Value:        &cacheName,
-	DefaultValue: "",
+	DefaultValue: []string{""},
```

Comment:
This seems like an odd initial value, I think it should either be `[]string{}` or `nil`

Reply:
I tried to use `[]string{}`, but that panic... I'll try `nil` then.

Reply:
Fixed now